mongorestore/mongorestore_test.go

```diff
@@ -1128,11 +1126,21 @@ func TestSkipStartAndAbortIndexBuild(t *testing.T) {
 			countBeforeRestore, err := dbLocal.Collection("oplog.rs").CountDocuments(ctx, bson.D{})
 			So(err, ShouldBeNil)
 
+			currentTS := uint32(time.Now().UTC().Unix())
```

Comment:
I think it would be better to get the timestamp from the latest oplog entry. Currently mongod and the test process are on the same host so it's probably fine. But if we ever want to run the tests on a remote host, there could be clock skew issues. So getting the timestamp from the oplog is a little more reliable and future-proof.

Reply:
I was getting a weird error when I was filtering with `currentTS`, so I changed the method of testing so that now I'm counting _all_ of the documents before and after restore.

Reply:
Okay, that works :+1: